### PR TITLE
improve ux

### PR DESCRIPTION
**Pull request by 0xtiki**
redirect to server page after login
filter poll list by user
change userId to discord user name

### CODE REVIEW

File: components/ThemedDateTimePicker.tsx [hunk 1]
```diff
@@ -37,6 +38,7 @@ const ThemedDateTimePicker: React.FC<ThemedDateTimePickerProps> = ({
         showTimeSelect
         dateFormat='MMMM d, yyyy h:mm aa'
         ref={ref}
+        minDate={new Date()}
```
Comment (wpmonty) [inline components/ThemedDateTimePicker.tsx:49]: passed days are disabled, but past time is still available. Maybe need to write a custom filter https://stackoverflow.com/questions/59826534/react-datepicker-mintime-and-maxtime-not-works

File: hooks/useGovernatorUser.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+export const useGovernatorUser = () => {
+
+    const [governatorUser, setGovernatorUser] = useAtom(governatorUserAtom);
+    const {data: session} = useSession();
+
+    const getGovernatorUser = useCallback(async () => {
+
+        const discordId = session?.discordId as string;
+        const userResponse = await privateBaseFetcher(
```
Comment (wpmonty) [inline hooks/useGovernatorUser.ts]: if `discordId`is falsey, should we avoid querying? or throw

File: pages/servers/[serverId]/polls/index.tsx [hunk 3]
```diff
@@ -55,8 +56,9 @@ const Polls: NextPage = () => {
   const router = useRouter()
   const { loading, currentServer } = useServers()
   const { channels, loading: isLoadingChannels } = useServer()
+  const governatorUser = useGovernatorUser()
 
-  const { data, error, mutate } = useSWR('/poll/list', privateBaseFetcher)
+  const { data, error, mutate } = useSWR(governatorUser.userId ? `/poll/user/${ governatorUser.userId }` : null, privateBaseFetcher);
```
Comment (wpmonty) [inline pages/servers/[serverId]/polls/index.tsx:61]: is swr cool with a null here?

File: components/ThemedDateTimePicker.tsx (hunk 1 above)
Comment (0xtiki) [reply components/ThemedDateTimePicker.tsx:49]: found your link, seems to work

File: pages/servers/[serverId]/polls/index.tsx (hunk 3 above)
Comment (0xtiki) [reply pages/servers/[serverId]/polls/index.tsx:61]: yes

File: components/ThemedDateTimePicker.tsx (hunk 1 above)
Comment (wpmonty) [reply components/ThemedDateTimePicker.tsx:49]: perfect

File: components/polls/PollForm.tsx [hunk 4]
```diff
@@ -156,7 +156,7 @@ const PollForm: React.FC<BoxProps> = ({ ...props }) => {
         poll_options: pollOptions,
         allow_options_for_anyone: false, // hardcoded as false for now
         single_vote: data.single_vote,
-        end_time: data.end_time,
+        end_time: data.end_time?.toISOString(),
```
Comment (wpmonty) [inline components/polls/PollForm.tsx]: If time is required, we need to enforce it. Can't be `?` optionally false.

**Review by wpmonty — COMMENTED** [review]
One small additional: "Block Height" label appears when default strategy is selected. It should be hidden with its field.

File: components/NavBar.tsx [hunk 5]
```diff
@@ -156,7 +156,7 @@ const NavBar = ({ waitlistDisabled }: { waitlistDisabled: boolean }) => {
                     <Link href='/servers'>
                       <a>
                         <Text as='span' fontSize='15px' fontWeight='500'>
-                          Servers
+                          Create Poll
```
Comment (wpmonty) [inline components/NavBar.tsx]: yeah maybe we call it "Dashboard" instead for now

**Review by wpmonty — APPROVED** [review]

**Pull request merged**